config/pkg/pldconf/privatetxmgr.go

```diff
@@ -58,4 +59,5 @@ type PrivateTxManagerSequencerConfig struct {
 	StaleTimeout                        *string `json:"staleTimeout,omitempty"`
 	RoundRobinCoordinatorBlockRangeSize *int    `json:"roundRobinCoordinatorBlockRangeSize,omitempty"`
 	AssembleRequestTimeout              *string `json:"assembleRequestTimeout,omitempty"`
+	BlockRangeSize                      *int64  `json:"blockRangeSize,omitempty"`
 }
```

Comment:
Not sure I understand this structure and where it fits (I see `SequencerConfig` in a dedicated file)